tests/src/test/scala/cats/tests/ApplicativeTests.scala

```diff
@@ -4,7 +4,7 @@ package tests
 import cats.Applicative
 
 
-class ApplicativeTests extends CatsSuite {
+class ApplicativeCheck extends CatsSuite {
```

Comment:
Not sure why this rename? I noticed that `ApplicativeErrorCheck` is also using the `Check` suffix. But that looks like an outlier? The vast majority have the suffix `Tests`.  There is also `MonadErrorSuite`, `BitraverseTest`, `CategoryTest`, `ComposeTest`, `ContravariantTest`, `FoldableCheck`, `ReducableCheck`, `FunctorTest`, `MonadTest`. If you leave  this one alone, I can submit another PR to unify them.